schemas/v1.0/azure.yaml.json

```diff
@@ -79,6 +79,11 @@
                         "type": "string",
                         "title": "Path to the service source code directory"
                     },
+                    "dotnetProjectFile": {
```

Comment:
@ellismg Should we repurpose `project` for this use case, i.e. `project` can either be a directory, or a path to a file. I'd personally want to see this done as a feature separate from the bug fix (let's just error out for now).

Trying to avoid language-specific schema unless we deem something that is truly specialized. @savannahostrowski may have thoughts from UX perspective as well.